src/node_internals.h

```diff
@@ -261,6 +261,10 @@ void FatalException(v8::Isolate* isolate,
                     v8::Local<v8::Value> error,
                     v8::Local<v8::Message> message);
 
+void InternalFatalException(v8::Isolate* isolate,
+                            v8::Local<v8::Value> error,
+                            v8::Local<v8::Message> message,
+                            bool from_promise);
```

Comment:
Doesn't need to be here, it's only used in node.cc.  Declare it `static` or `inline` in that file.